Assets/MixedRealityToolkit-SDK/Features/Boundary/Components/Scripts/FloorPlaneVisualizer.cs

```diff
@@ -0,0 +1,83 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Microsoft.MixedReality.Toolkit.Internal.Definitions.BoundarySystem;
+using Microsoft.MixedReality.Toolkit.Internal.Managers;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.SDK.Boundary
+{
+    /// <summary>
+    /// Visualizes the floor plane.
+    /// </summary>
+    public class FloorPlaneVisualizer : MonoBehaviour
+    {
+        [Tooltip("Should the floor plane be visualized?")]
+        [SerializeField]
+        private bool isFloorPlaneVisualized = true;
+
+        /// <summary>
+        /// Gets or sets a value indicating whether or not the floor plane is / should be visualized.
+        /// </summary>
+        public bool IsFloorPlaneVisualized
+        {
+            get { return isFloorPlaneVisualized; }
+            set
+            {
+                if (value != isFloorPlaneVisualized)
+                {
+                    isFloorPlaneVisualized = value;
+                }
+            }
+        }
+
+        private MixedRealityBoundaryVisualizationProfile visualizationProfile;
+
+        /// <summary>
+        /// The <see cref="GameObject"/> (Quad) used to visualize the play area.
+        /// </summary>
+        private GameObject floorPlane = null;
```

Comment:
Shouldn't this component be added to the GameObject we want to visualize? (or do we do that below?)

Reply:
it gets added to the game object that has the script attached.